src/tools/pub/publishing.md

```diff
@@ -41,6 +41,13 @@ are a few additional requirements for uploading a package:
   and Git dependencies don't support version resolution as well as hosted
   dependencies do.
 
+* You must have a [Google Account,][Google Account]
+  which pub uses to manage package upload permissions.
+  Your Google Account can be associated with a Gmail address or
+  with any other email address.
```

Comment:
>   Your Google Account can be associated with a Gmail address or
  with any other email address.

This goes without saying.

Reply:
It’s not clearly documented anywhere I can find, and I think a lot of people assume that Google Account == gmail.com or google.com.

Reply:
Okay, I'm not concern with this being pointed out.